index.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="welcome-container">
       <h1 class="welcome-container__headline">Welcome</h1>
-      <p class="welcome-container__paragraph">TODO: General description of how the website works</p>
+      <p class="welcome-container__paragraph"></p>
```

Comment:
You can probably remove this if you don't plan on writing a description :D

Reply:
Fixed it in the branch :D May it be that you didn't switch on my branch by mistake? 😁

Reply:
Which branch? The branch of this PR still has the empty paragraph :)

Reply:
lol